### PR TITLE
[TA] Migrate azure-json for those models introduced at v1 version  

### PR DESCRIPTION
**Pull request by mssfang**
TA's history:

Before consolidation, Text Analytics uses a different spec to generate the models and the SDK.
After consolidation done, those models no longer automate generated models but to become "hand-written" models that require manual changes. Which is the reason we have missed at previous migration [PR](https://github.com/Azure/azure-sdk-for-java/pull/37238).

This PR finishes the TA azure-json migration from jackson-databind dependency.

### CODE REVIEW

File: sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/EntityCertainty.java [hunk 1]
```diff
@@ -44,7 +43,6 @@ public EntityCertainty() {
      * @param name The string name to look for.
      * @return The corresponding {@code EntityCertainty}.
      */
-    @JsonCreator
```
Comment (samvaity) [inline sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/EntityCertainty.java:47]: Did this not need revapi to be updated?

File: sdk/textanalytics/azure-ai-textanalytics/assets.json [hunk 2]
```diff
@@ -2,5 +2,5 @@
   "AssetsRepo": "Azure/azure-sdk-assets",
   "AssetsRepoPrefixPath": "java",
   "TagPrefix": "java/textanalytics/azure-ai-textanalytics",
-  "Tag": "java/textanalytics/azure-ai-textanalytics_fbd1568119"
+  "Tag": "java/textanalytics/azure-ai-textanalytics_15b7c8efde"
```
Comment (samvaity) [inline sdk/textanalytics/azure-ai-textanalytics/assets.json:5]: We should do LiveTests to validate the changes.

**Review by samvaity — APPROVED** [review]

File: sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/EntityCertainty.java (hunk 1 above)
Comment (mssfang) [reply sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/EntityCertainty.java:47]: It doesn't.

**Pull request merged**